modules/ROOT/pages/audit-logging.adoc

```diff
@@ -2099,13 +2099,29 @@ You can access the
 https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/[Audit Log REST API] from the Audit Logging Query API and its
 https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/organizations/8bfc8bbf-5508-419e-aadc-77dfe18a8172/assets/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/1.0.5/files/fat-raml/zip[RAML].
 
+
 === Use a Query Loop to make Requests in Audit Logs
 
-To gather data, you can initiate query loops and use cursors to navigate paginated audit logs. To avoid performance degradation, you can stop the query loop using the `total` by using the value returned by `IncludeTotal`.
+To query audit logs, the current https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/minor/1.0/console/method/%23245/ [query api] will continue to work.
+Pagination in that api is via offset in the query payload, and the response returns the total number of records.
```

Comment:
Same with this. I am not really sure what this means.

Reply:
The old api uses offset pagination.
The new api format user query pagination.
So calling out the old method of pagination so we can explain that the new method uses a different pagination.

Reply:
```suggestion
```

Reply:
Proposing removing this line and consolidating this information into the above paragraph. Let me know what you think.